modules/auxiliary/scanner/karaf/login.rb

```diff
@@ -0,0 +1,158 @@
+##
+# This module requires Metasploit: http://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+require 'msf/core'
+require 'net/ssh'
+require 'metasploit/framework/login_scanner/ssh'
+require 'metasploit/framework/credential_collection'
+
+class Metasploit3 < Msf::Auxiliary
+
+  include Msf::Auxiliary::Report
+  include Msf::Auxiliary::CommandShell
+  include Msf::Auxiliary::AuthBrute
+  include Msf::Auxiliary::Scanner
+
+  def initialize
+    super(
+      'Name'        => 'Karaf Default Credential Scanner',
+      'Description' => %q{
+        This module uses default Karaf credentials to login to the console via ssh.
+      },
+      'Author'      => [
```

Comment:
We do Karaf parties every Thursday night, @wchen-r7 and @wvu-r7, you are both invited. :wine_glass: